csrankings.csv

```diff
@@ -18164,3 +18164,36 @@ eva Kühn,TU Wien,http://www.complang.tuwien.ac.at/eva,NOSCHOLARPAGE
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,NTNU,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Shizhe Zhou,Hunan University,https://shizhezhou.github.io/,NOSCHOLARPAGE
+Guo Chen,Hunan University,https://1989chenguo.github.io/,ouriG7cAAAAJ
+Zhe Quan,Hunan University,http://csee.hnu.edu.cn/people/quanzhe,NOSCHOLARPAGE
+Hao Chen0002,Hunan University,http://www.aimlab.org/haochen/,NOSCHOLARPAGE
+Xiangxiang Zeng,Hunan University,https://xzenglab.github.io/xzeng/,B20HBMIAAAAJ
+Kenli Li,Hunan University,http://csee.hnu.edu.cn/people/likenli,NOSCHOLARPAGE
+Di Wu0002,Hunan University,https://www.ics.uci.edu/~dwu3/,NOSCHOLARPAGE

```

Comment:
Missing space between the name and the suffix (this is one of several such cases).

Reply:
Ahhh, thank you for your reminding. I have fixed that problem. Please check again. Thank you.